docs/connectors/taps/mixpanel.rst

```diff
@@ -0,0 +1,111 @@
+
+.. _tap-mixpanel:
+
+Tap Mixpanel
+------------
+
+Authentication
+''''''''''''''
+
+The Mixpanel API uses Basic Authorization with the ``api_secret`` from the tap config in base-64 encoded format.
+It is slightly different than normal Basic Authorization with username/password. All requests include this
+header with the ``api_secret`` as the username, with no password.
+
+* Authorization: ``Basic <base-64 encoded api_secret>``
+
+More details may be found in the Mixpanel `API Authentication <https://developer.mixpanel.com/docs/data-export-api#section-authentication>`_
+instructions.
+
+The API secret can be found in the Mixpanel Console, upper-right Settings (gear icon),
+Organization Settings > Projects and in the Access Keys section. For this tap,
+only the ``api_secret`` is needed (the ``api_key`` is legacy and the token is used only for uploading data).
+
+.. warning::
+
+    Each Mixpanel project has a different ``api_secret``; therefore each Mixpanel pipeline instance is for a single project.
+
+
+Configuring what to extract
+'''''''''''''''''''''''''''
+
+PipelineWise configures every tap with a common structured YAML file format.
+A sample YAML for Jira replication can be generated into a project directory by
```

Comment:
maybe the JIRA here is not what you meant ?
`Jira replication`

Reply:
fixed by 95ea1211bd85b38d088899befbc791ec124acbd1